lib/clarkson-core-objects.php

```diff
@@ -54,22 +38,14 @@ public function get_term( $term ) {
 	/**
 	 * Get users by id.
 	 *
-	 * @param \WP_User[]|int[] $users WP_User object (or deprecated user id).
+	 * @param \WP_User[] $users array of \WP_User objects.
 	 *
 	 * @return \Clarkson_User[]
 	 */
-	public function get_users( $users ) {
+	public function get_users( array $users ): array {
 		$user_objects = array();
 
 		foreach ( $users as $user ) {
-			if ( ! $user instanceof \WP_User ) {
-				_doing_it_wrong( __METHOD__, 'Deprecated get_user called with an ID. Supply a \WP_User object or use \'Clarkson_User::get(user_id)\' instead.', '1.0.0' );
-				$user_id = $user;
-				$user    = get_userdata( $user_id );
-				if ( ! $user instanceof \WP_User ) {
-					throw new Exception( $user_id . ' does not exist' );
-				}
-			}
 			$user_objects[] = $this->get_user( $user );
```

Comment:
Moeten we hier nog wel een guard voor instanceof WP_User hebben? omdat we anders een array gevuld met `null`etjes kunnen krijgen?

Reply:
Als het geen WP_User is slaan we hem over?

Reply:
Of we doen het wel zo, maar filteren daarna de null waardes uit de array?

Reply:
get_user() returned geen null. Hij pakt hier voor zover ik weet de standaard WordPress functionaliteit waarbij je dan een \WP_User object hebt waar niks in zit. Als we dit gaan filteren, moeten we op get_user() ook null kunnen teruggeven als de user niet bestaat.

Reply:
Zie #184. Onderdeel daarvan wordt dat we bij ongeldige object geen throws meer doen, maar null teruggeven.

Reply:
Hij slaat nu dingen die geen instance zijn van WP_User over.